Documentation/openshiftdocs/modules/remotewritespec.adoc

```diff
@@ -35,7 +35,7 @@ link:prometheusrestrictedconfig.adoc[PrometheusRestrictedConfig]
 
 |oauth2|*monv1.OAuth2|Defines OAuth2 authentication settings for the remote write endpoint.
 
-|proxyUrl|string|Defines an optional proxy URL. It is superseded by the cluster-wide proxy, if enabled.
+|proxyUrl|string|Defines an optional proxy URL. It is superseded by the cluster-wide proxy when it is enabled, prioritizing the HTTPS proxy.
```

Comment:
Hello, I like the change you are suggesting, I believe it is just missing the information about the replacement there. So I would suggest something like:

```suggestion
|proxyUrl|string|Defines an optional proxy URL. If the cluster-wide proxy is enabled, it replaces the proxyUrl setting. The cluster-wide proxy supports both HTTP and HTTPS proxies, with HTTPS taking precedence.
```
I hope this is correct. I was not sure if the "it" in your suggested text refers to the cluster-wide proxy or the proxyURL property :sweat_smile:

Reply:
much clearer, thanks.